.scalafmt.conf

```diff
@@ -0,0 +1,4 @@
+version = 2.7.5
+align = more // For pretty alignment.
+maxColumn = 120
+newlines.alwaysBeforeTopLevelStatements = true
```

Comment:
Does `scalafmtAll` breaks everything? :/ I'd expect it to refactor 90% of our files.

Reply:
It does -- I mostly just wanted to get it in here, but have it be unused. But, I think it would be better to spend a bit of time aligning the rules with the existing code before merging. Putting PR in draft.

Reply:
There is this tool that hopefully can generate the scalafmt file that would fit our needs: https://github.com/tanishiking/scalaunfmt 

(I would not be that positive about it), scalafmt is definitely going to break our type params with lots of imports formatting ):

Reply:
Great, I'll take a look at that tool.  I think it's more important to be consistent within the codebase than adhere to some arbitrary external standard, so I'll try to write the rules that way.